api/src/main/java/org/pmiops/workbench/db/dao/RdrExportDao.java

```diff
@@ -10,24 +10,34 @@
 
 public interface RdrExportDao extends CrudRepository<DbRdrExport, Long> {
 
-  static final String WORKSPACE_IDS_TO_EXPORT_QUERY =
-      "select w.workspace_id from  workspace w LEFT JOIN "
+  String USER_IDS_TO_EXPORT_QUERY =
+      "select u.user_id from user u LEFT JOIN "
+          + "rdr_export rdr on w.workspace_id = rdr.entity_id and rdr.entity_type = 1 where "
```

Comment:
w.workspace_id seems wrong? Do you mean u.user_id?

Reply:
Great catch, thanks. I made a bunch of changes to the tests so that they actually exercise the code rather than mocking everything. Added a regression test which covered this case.